### PR TITLE
fix(comment-input): Fix the input text being chopped up

### PR DESCRIPTION
**Pull request by machour**
| Question         | Response    |
| ---------------- | ----------- |
| Version?         | master     |
| Devices tested?  | iPhone 7, Android Nexus (simulators) |
| Bug fix?         | yes     |
| New feature?     | no      |
| Includes tests?  | no      |
| All Tests pass?  | yes     |
| Related ticket?  | #536       |

## Description

Android had a flickering problem when height started with 0.
Also adapted the height a bit more for the final input.

### CODE REVIEW

**Comment by chinesedfan** [comment]
@machour So the culprit is the zero height? And the height adjustment is for better UX?

**Comment by machour** [comment]
@chinesedfan It's quiet confusing, my bad.

Originally, I used the inspector to see the height on Android, and found it to be 37.
When I set it, I noticed an improvement, but further testing showed some weird cases where the bug would still happen.

By changing the initial height after that, I was not able to reproduce it anymore, and I figured I'll keep that 37 anyway as it is the actual result.

File: src/components/comment-input.component.js [hunk 1]
```diff
@@ -51,6 +51,11 @@ const StyledPostButtonIcon = styled(PostButtonIcon)`
   color: ${props => (props.disabled ? colors.grey : colors.primaryDark)};
 `;
 
+const inputMaxHeight = Platform.select({
```
Comment (chinesedfan) [inline src/components/comment-input.component.js]: Should be renamed as `inputMinHeight`?

Comment (machour) [reply src/components/comment-input.component.js]: Totally! Got fooled by Math.max() 😸

**Comment by machour** [comment]
@chinesedfan fixed, thanks!

**Pull request merged**